UKHomeOffice/sgmr-service
# SPCR-159 Add descriptive error messages to the Voyage form

## Description
Error messages displayed above the question should also appear in an an error summary under 'There is a problem'. When a user clicks on an error in the summary it should scroll the page the the error. 

### To test
- Go to the dashboard
- Click on start now
- On each page, leave inputs empty and click save and continue
- The page should scroll an error summary with a list of errors
- (note: on arrival and departure pages the port input is currently causing errors but this should be resolved with the new ref data)
- When an error link is clicked, it should scroll to the relevant error in the form.
- Selecting/typing in an input field should remove the errors 


(This also needs to be manually tested as problems could have been missed by myself)

\* Required






Ensure you delete the branch once the Pull Request is merged.
